ckks/encryptor.go

```diff
@@ -210,84 +203,114 @@ func (encryptor *pkEncryptor) EncryptFromCRPFastNew(plaintext *Plaintext, crp *r
 // encrypt with sk: ciphertext = [-a*sk + m + e, a]
 func (encryptor *pkEncryptor) encrypt(plaintext *Plaintext, ciphertext *Ciphertext, fast bool) {
 
+	lvl := plaintext.Level()
```

Comment:
Which behaviour?

Reply:
What I refer to as "the behavior" is that the encryption function will use the provided plaintext level and produce a ciphertext at the same level. Since this is rather implicit (no explicit "level" parameter is given to the encryption functions), it needs to be abundantly documented.

Hence, the same line as the one added on the `Encryptor` interface type (perhaps slightly adapted) should also be added on the godoc of each interface method.

Reply:
Ok, this is now documented.